adaptivecards-hostconfig.json

```diff
@@ -1,174 +1,121 @@
 ﻿{
   "supportsInteractivity": true,
-  "strongSeparation": {
-    "spacing": 40,
+  "spacing": {
+    "small": 4,
+    "default": 8,
+    "medium": 16,
+    "large": 24,
+    "extraLarge": 32,
+    "padding": 8
+  },
+  "separator": {
     "lineThickness": 1,
-    "lineColor": "#eeeeee"
+    "lineColor": "#cccccc"
   },
   "fontFamily": "\"Segoe UI\", sans-serif",
   "fontSizes": {
     "small": 12,
-    "normal": 13,
+    "default": 13,
     "medium": 15,
     "large": 17,
     "extraLarge": 19
   },
   "fontWeights": {
     "lighter": 200,
-    "normal": 400,
+    "default": 400,
     "bolder": 700
   },
-  "colors": {
-    "dark": {
-      "normal": "#000000",
-      "subtle": "#808c95"
-    },
-    "light": {
-      "normal": "#ffffff",
-      "subtle": "#88ffff"
-    },
-    "accent": {
-      "normal": "#2e89fc",
-      "subtle": "#802E8901"
-    },
-    "attention": {
-      "normal": "#ffd800",
-      "subtle": "#CCFFD800"
-    },
-    "good": {
-      "normal": "#00ff00",
-      "subtle": "#CC00FF00"
+  "containerStyles": {
+    "default": {
+      "backgroundColor": "#00000000",
+      "foregroundColors": {
+        "default": {
+          "default": "#000000",
+          "subtle": "#808c95"
+        },
+        "accent": {
+          "default": "#2e89fc",
```

Comment:
adaptivecards beta09 still seem to use "normal" instead of "default". It was changed in later versions of adaptive cards. Colours in current scheme will be defaulted to black as as properties will not match.